### PR TITLE
added twitter script to the site page to better handle social media b…

### PR DESCRIPTION
**Pull request by tmccoy529**
This is a script that builds the twitter and facebook href to be posted to the corresponding social media account. This is being added her because social media buttons are now going to be a component inside of the library and this alleviates the need to add code snippets to SE pages.

### CODE REVIEW

File: src/site/SiteNavigation.js [hunk 1]
```diff
@@ -226,3 +226,36 @@ document.addEventListener("DOMContentLoaded", onDocumentReady);
  * https://gomakethings.com/listening-for-click-events-with-vanilla-javascript/
  */
 document.addEventListener("click", handleDocumentClick, false);
+
+/**
```
Comment (martypowell) [inline src/site/SiteNavigation.js:230]: just curious, why did you choose to put this in the sitenavigation file? I would have expected this to be placed in https://github.com/baltimorecounty/dotgov-javascript/blob/integration/src/site/Site.js or probably the better solution would be to create a file in site called SocialMedia.js (or something like that)?

Comment (martypowell) [reply src/site/SiteNavigation.js:230]: @tmccoy529

Comment (tmccoy529) [reply src/site/SiteNavigation.js]: hahaha it was late and I was trying to get it done and tested. But you are correct, it should be in its own socialmedia file. This has been changed

Comment (martypowell) [reply src/site/SiteNavigation.js]: @tmccoy529 looks good, but i still think you need to import it into Site.js?

Comment (tmccoy529) [reply src/site/SiteNavigation.js]: Hahahah look man, reading comprehension isnt my strong suit.....this has now been imported into site.js

**Pull request merged**